pinot-common/pom.xml

```diff
@@ -107,6 +107,11 @@
     </plugins>
   </build>
   <dependencies>
+    <dependency>
+      <groupId>io.prometheus.jmx</groupId>
+      <artifactId>jmx_prometheus_javaagent</artifactId>
+      <version>0.19.0</version>
```

Comment:
Shall we define the versions in the dependency management in the root pom?

Reply:
Ah sure!

Reply:
Since it is declared in the plugin, do we still need to import it here?

Reply:
We need to, as we're no running it as an agent in the test. It's being used as a normal maven dependency.